gethrpc/subscription_test.go

```diff
@@ -19,6 +19,7 @@ package rpc
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/ethereum/go-ethereum/common/hexutil"
```

Comment:
-.-
Can you use https://golang.org/pkg/encoding/hex/ or at least some util from polkadot repo